unit_tests/UnitTestCreateOnDevice.C

```diff
@@ -10,39 +10,23 @@
 #include <KokkosInterface.h>
 #include "utils/CreateDeviceExpression.h"
```

Comment:
Is this header file still needed since the test has changed to not keep a class device pointer.

Reply:
@overfelt The class device pointer was never used in the test. Instead a new device pointer was created in the test function. This section requires that header file

https://github.com/Exawind/nalu-wind/blob/b70bdfee832da7ce82611be19f2d4ccbace11f45/unit_tests/UnitTestCreateOnDevice.C#L69-L76